### DISCUSSION TITLE
How can I check if there are any entity at specific location?

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#750] [Help] [answered]
Hi. I want to check, if there are any entity at specific location. How can I do it?

**Comment by MrlnHi** [comment, marked as answer]
I probably over engineered this but it should work:
```java
Pos pos = new Pos(0, 64, 0);
Collection<Entity> entities = instance.getEntityTracker().chunkEntities(pos, EntityTracker.Target.ENTITIES);
Set<Entity> entitiesAtPos = entities.stream().filter(entity -> entity.getPosition().sameBlock(pos)).collect(Collectors.toSet());
```

**Comment by emortaldev** [comment]
I know this has already been answered, but what is the purpose? You may want to check bounding boxes instead and see if they intersect with the block or the player

**Reply by ghost** [reply]
Thank you for the suggestion, I’ll also try it
